public/js/createTemplate.js

```diff
@@ -111,36 +118,36 @@ function addList(){
     inputs.append(newDiv);
 }
 
-function addComments(){
-    const commentsList = document.getElementsByName('comments_list_id')
-    if(commentsList.length === 0){
-        let inputs = document.getElementById('inputs')
-        let hiddenInputs = document.getElementById('hiddenInputs')
-
-        let newDiv = document.createElement('div')
-        newDiv.className = 'form-group col-md-4'
-
-        const label = document.createElement('label');
-        label.textContent = 'Kies een opmerking lijst'
-
-        let select = document.createElement('select');
-        select.className = 'form-select';
-        select.name = 'comments_list_id';
-        dynamicLists.forEach(dynamicList => {
-            if(!dynamicList.is_main_list){
-                const option = document.createElement('option');
-                option.value = dynamicList.id;
-                option.textContent = dynamicList.name;
-                select.append(option);
-            }
-        });
-        let deleteButton = document.createElement('button')
-        deleteButton.className = 'float-right btn btn-danger'
-        deleteButton.textContent = 'X'
-        deleteButton.addEventListener('click', function () {
-            newDiv.parentNode.removeChild(newDiv)
-        })
-        newDiv.append(label,deleteButton, select);
-        inputs.append(newDiv);
-    }
-}
+// function addComments() {
+//     const commentsList = document.getElementsByName('comments_list_id')
+//     if (commentsList.length === 0) {
+//         let inputs = document.getElementById('inputs')
+//         let hiddenInputs = document.getElementById('hiddenInputs')
+//
+//         let newDiv = document.createElement('div')
+//         newDiv.className = 'form-group col-md-4'
+//
+//         const label = document.createElement('label');
+//         label.textContent = 'Kies een opmerking lijst'
+//
+//         let select = document.createElement('select');
+//         select.className = 'form-select';
+//         select.name = 'comments_list_id';
+//
+//         dynamicLists.forEach(dynamicList => {
+//             const option = document.createElement('option');
+//             option.value = dynamicList.id;
+//             option.textContent = dynamicList.name;
+//             select.append(option);
+//         });
+//
+//         let deleteButton = document.createElement('button')
+//         deleteButton.className = 'float-right btn btn-danger'
+//         deleteButton.textContent = 'X'
+//         deleteButton.addEventListener('click', function () {
+//             newDiv.parentNode.removeChild(newDiv)
+//         })
+//         newDiv.append(label, deleteButton, select);
+//         inputs.append(newDiv);
+//     }
+// }
```

Comment:
Weet niet zeker of je dit erbij wilt hebben in de pull request of niet, of dat dit nog later word omgebouwd